.github/workflows/reusable_docker_build.yml

```diff
@@ -27,14 +27,21 @@ on:
 
 permissions: {}
 
+concurrency:
+  group: 'docker_build-${{ github.ref }}'
+  cancel-in-progress: true
+
```

Comment:
I removed this because it causes issues when using this in multiple workflows - e.g. in the previous setup we had a separate workflow for building firedrake-parmmg and firedrake-um2n if both get triggered in a PR, it would randomly cancel one of them because they are workflows with the same github ref and thus the same group. I think it's fine to add it back with our current use of it in a single workflow  - but note that it's already specified in docker_firedrake-parmmg.yml

Reply:
Ahh okay. Reverted in https://github.com/mesh-adaptation/docs/pull/159/commits/ca08eece5e1a178d123f1c929a479804b5ee6d7e.